spec/models/recipe_spec.rb

```diff
@@ -20,9 +20,24 @@
       end
     end
   end
+
+  describe  'observing recipes relationship' do
+    context 'with ingredients' do
+      it 'has ingredients' do
+      recipe = create(:recipe, name: subject.name)
+      ingredient = Ingredient.new(name: 'Ingredient')
+      ingredient2 = Ingredient.new(name: 'Ingredient 2')
+      used_ingredient = UsedIngredient.new(recipe_id: recipe.id, ingredient_id: ingredient.id)
+      used_ingredient2 = UsedIngredient.new(recipe_id: recipe.id, ingredient_id: ingredient2.id)
+      expect(recipe).to be_valid
```

Comment:
I would hope for a more useful assertion here, something like expect(recipe.ingredients.length).to eq(2)